dask/dataframe/reshape.py

```diff
@@ -151,11 +149,10 @@ def get_dummies(
         if not all(has_known_categories(data[c]) for c in columns):
             raise NotImplementedError(unknown_cat_msg)
 
-    package_name = data._meta.__class__.__module__.split(".")[0]
-    dummies = sys.modules[package_name].get_dummies
+    xd = get_meta_library(data)
 
     return map_partitions(
-        dummies,
+        xd.get_dummies,
```

Comment:
Similar comment: Seems like it should be fine to just use `get_meta_library(data).get_dummies` in the call to `map_partitions`?